hazelcast/src/main/java/com/hazelcast/nio/tcp/TcpIpAcceptor.java

```diff
@@ -283,7 +286,6 @@ public void run() {
 
         private void configureAndAssignSocket(Channel channel) {
             try {
-                ioService.configureSocket(channel.socket());
```

Comment:
Again, the socket is now configured later in `channel.start()`, right?

Reply:
That is correct.